From 2b2959a40f8f92188559cbb4ec6376c5fbe13bb2 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:49:58 -0400
Subject: [tools] perf/evlist: Remove unnecessary parentheses

Message-id: <1408953210-26343-111-git-send-email-jolsa@redhat.com>
Patchwork-id: 88859
O-Subject: [PATCH RHEL7.1 BZ1133083 110/322] perf evlist: Remove unnecessary parentheses
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 2bcab6c146fde0d0286b132068111f98b6217460
Author: Adrian Hunter <adrian.hunter@intel.com>
Date: Mon Dec 9 15:18:37 2013 +0200
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/evlist.c b/tools/perf/util/evlist.c
index 7bb6ee1..4d0945c 100644
--- a/tools/perf/util/evlist.c
+++ b/tools/perf/util/evlist.c
@@ -732,7 +732,7 @@ static long parse_pages_arg(const char *str, unsigned long min,
    return -EINVAL;
  }
 
- if ((pages == 0) && (min == 0)) {
+ if (pages == 0 && min == 0) {
   /* leave number of pages at 0 */
  } else if (pages < (1UL << 31) && !is_power_of_2(pages)) {
   /* round pages up to next power of 2 */
-- 
1.7.1